cpp/src/binaryop/jit/code/traits.cpp

```diff
@@ -43,10 +43,17 @@ R"***(
     template <typename Type>
     constexpr bool isFloat = false;
 
+    template <typename T>
+    constexpr bool is_timestamp_t_v =
```

Comment:
This trait already exists in our `traits.hpp`.

Furthermore, the canonical way of writing this is `is_timestamp_v`.

Reply:
i'll change the constexpr to `is_timestamp_v`

the jit code presently only includes `types.hpp` and `timestamps.hpp` to compile and does *not* include `traits.hpp` and hence i defined that in the jit traits header. the first 2 headers have only minimal includes. bringing in the `traits.hpp` will also bring in a bunch of other includes and may require changes to the jit compilation include directives (and jit types etc.), which i was trying to avoid.